generated/nidigital/nidigital/unit_tests/test_nidigital.py

```diff
@@ -1,6 +1,7 @@
 import _mock_helper
 
 import nidigital
+from nidigital.enums import PinState
```

Comment:
Stylistically I hadn't seen us import a specific enum into our namespace elsewhere in this codebase. Do we want to teach this?

Reply:
I agree that it's probably better stylistically not to pollute the namespace.
For what it's worth, [we already do this in the system tests.](https://github.com/ni/nimi-python/blob/7911a1400de731e03b620ee9492599aa19f67dcc/src/nidigital/system_tests/test_system_nidigital.py#L10)

If you want me to change it, though, I will.

Reply:
I did it that way to keep this [monstrosity](https://github.com/ni/nimi-python/blob/7911a1400de731e03b620ee9492599aa19f67dcc/src/nidigital/system_tests/test_system_nidigital.py#L405) readable. But I agree that we should be consistent stylistically. Change it, I'll do the same in system test

Reply:
Done.